nix/modules/fedimintd.nix

```diff
@@ -124,6 +129,23 @@ let
             '';
           };
         };
+        api_iroh = {
```

Comment:
Maybe instead of new options, we should just open all sets of ports on existing settings?

Reply:
I first just tried to mirror the existing env var semantics, but I think only allowing either iroh or ws would make a lot more sense. Idk if that would work though or if we still read the ws env vars even if iroh is active.